js/src/dapps/registry/accounts/accounts.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+
+// import styles from './lookup.css';
+
+const renderAccount = (active) => (account) => (
```

Comment:
Should be a state-less React component instead to avoid re-rendering:
```js
const RenderAccount = ({active, account}) => ( ... );
```
and usage
```js
Object.values(all).map(account => (<RenderAccount account={account} active={selected} /> ));
```